lib/travis/build/bash/travis_export_go.bash

```diff
@@ -0,0 +1,10 @@
+travis_export_go() {
+  if [[ ! "${1}" ]]; then
+    echo 'Missing go version positional argument' >&2
+    return 86
+  fi
+  export TRAVIS_GO_VERSION="${1}"
+  export GIMME_GO_VERSION="${TRAVIS_GO_VERSION}"
+  : "${GOMAXPROCS:=2}"
```

Comment:
Would it be worth it to default to `nproc` rather than a hardcoded 2, at least on `$TRAVIS_OS_NAME == linux`?